runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/ArtifactServiceStagerTest.java

```diff
@@ -45,10 +45,13 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
 
 /**
  * Tests for {@link ArtifactServiceStager}.
  */
+@RunWith(JUnit4.class)
```

Comment:
This change seems unrelated. Can you break it into a separate commit?

Reply:
I can, just going to wait for the tests to pass on this giving me the green signal before.